src/Agent.Worker/Release/ReleaseJobExtension.cs

```diff
@@ -154,9 +154,13 @@ private IList<AgentArtifactDefinition> GetReleaseArtifacts(IExecutionContext exe
             IList<AgentArtifactDefinition> agentArtifactDefinitions)
         {
             Trace.Entering();
+            string commitsWorkFolder = String.Empty;
 
-            Trace.Info("Creating commit work folder");
-            string commitsWorkFolder = GetCommitsWorkFolder(executionContext);
+            if (agentArtifactDefinitions.Where(x => x.ArtifactType == AgentArtifactType.Jenkins).Any())
+            {
+                Trace.Info("Creating commit work folder");
```

Comment:
You could just use Any clause instead of using where clause and any clause on top of it.